src/Mono.Android/Profiles/api-25.xml.in

```diff
@@ -180835,12 +180835,6 @@
       </method>
       <method abstract="false" deprecated="not deprecated" final="false" name="entrySet" native="false" return="java.util.Set&lt;java.util.Map.Entry&lt;K, V&gt;&gt;" static="false" synchronized="false" visibility="public">
       </method>
-      <method abstract="false" deprecated="not deprecated" final="false" name="put" native="false" return="V" static="false" synchronized="false" visibility="public">
```

Comment:
I cannot exactly point to which api-xml-adjuster change caused that but I can say that it is about methods overrides and didn't bring API breakage in the resulting Mono.Android.dll.